Library/Homebrew/extend/os/linux/keg_relocate.rb

```diff
@@ -18,15 +17,15 @@ def relocate_dynamic_linkage(relocation)
   def change_rpath(file, old_prefix, new_prefix)
     return if !file.elf? || !file.dynamic_elf?
 
-    patchelf = DevelopmentTools.locate "patchelf"
-    cmd_rpath = [patchelf, "--print-rpath", file]
-    old_rpath = Utils.popen_read(*cmd_rpath, err: :out).strip
-
-    # patchelf requires that the ELF file have a .dynstr section.
-    # Skip ELF files that do not have a .dynstr section.
-    return if ["cannot find section .dynstr", "strange: no string table"].include?(old_rpath)
-    unless $CHILD_STATUS.success?
-      raise ErrorDuringExecution.new(cmd_rpath, status: $CHILD_STATUS, output: [[:stderr, old_rpath]])
+    patchelf = ::PatchELF::Patcher.new(file, logging: false)
+    # DT_RPATH is deprecated, using it here as the old code force used it
+    patchelf.use_rpath!
```

Comment:
`DT_RPATH` is much preferred over `DT_RUNPATH` because `DT_RUNPATH` can be overridden by `LD_LIBRARY_PATH`, and that can often cause breakage and a frequent source of support requests from users. `DT_RPATH` cannot be overridden by `LD_LIBRARY_PATH`.

Reply:
we continue using  DT_RPATH ?